### PR TITLE
Coverage

### LINKED ISSUE #36: Testing
Our test coverage is currently way too low so something needs to be done about this. Chapter 8 in Phoenix Framework covers testing so we will have a better grasp of how to improve our coverage. 

I'm currently on Chapter 6 so will have covered the testing chapter by Monday. 

- [x] read Chapter 8 - Phoenix Framework
- [ ] implement testing & improve coverage

### PR DESCRIPTION
**Pull request by samhstn**
Increases coverage to 100%

Fixes #36

### CODE REVIEW

**Review by nelsonic — COMMENTED** [review]
@shouston3 / @jackcarlisle well done for bumping up the test coverage. ⭐️ 
Keep up the great work! 🚀

**Comment by nelsonic** [comment]
is **coveralls** the only service we can use for a Phoenix project? 🤔

**Comment by samhstn** [comment]
Codecov was working before, I'll revoke access to coveralls and try to get codecov working again

File: test/support/test_helpers.ex [hunk 1]
```diff
@@ -0,0 +1,17 @@
+defmodule Skillswheel.TestHelpers do
+  alias Skillswheel.Repo
+
+  def insert_user(attrs \\ %{}) do
+    changes = Dict.merge(%{
+      name: "Some User",
+      username: "user #{Base.encode16(:crypto.rand_bytes(7))}",
```
Comment (jackcarlisle) [inline test/support/test_helpers.ex]: we don't have `username` in our users table. We can look at adding this if necessary but email, name and password should be fine for now

**Review by jackcarlisle — CHANGES_REQUESTED** [review]
Looks good! Will merge after `username` is removed from test helpers

**Review by jackcarlisle — APPROVED** [review]

**Comment by jackcarlisle** [comment]
👍 nice one! Merging

**Pull request merged**